src/migrations/1747997882050_add-communities.ts

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import { MigrationBuilder, ColumnDefinitions, PgType } from 'node-pg-migrate'
+
+export const shorthands: ColumnDefinitions | undefined = undefined
+
+export async function up(pgm: MigrationBuilder): Promise<void> {
+  pgm.createTable('communities', {
+    id: {
+      type: PgType.UUID,
+      primaryKey: true,
+      notNull: true
+    },
+    name: {
+      type: PgType.VARCHAR,
+      notNull: true
+    },
+    description: {
+      type: PgType.TEXT,
+      notNull: true
+    },
+    owner_address: {
+      type: PgType.VARCHAR,
+      notNull: true
+    },
+    thumbnail_url: {
+      type: PgType.VARCHAR,
+      notNull: false
+    },
+    places: {
+      type: 'varchar[]',
+      notNull: true,
+      default: '{}'
+    },
```

Comment:
If I remember correctly, places will be a list of places, associated with the community, that can be added, removed and probably sorted, paginated and  such. To allow us to perform those tasks, what do you think about eventually (whenever needed) a new table that will be linked to the collection and let us perform those operations?